src/e2e/step-definitions/zaak.ts

```diff
@@ -191,59 +200,54 @@ When(
     } else {
       throw new Error("No case number found");
     }
-
-    await this.page.waitForTimeout(1000);
   },
 );
 
 Then(
-  "Employee {string} sees the task assigned by Employee {string} in the newly created zaak tasks list",
+  "Employee {string} sees the task assigned to them by Employee {string} in the newly created zaak tasks list",
   { timeout: ONE_MINUTE_IN_MS },
-  async function (this: CustomWorld, user1, user2) {
+  async function (this: CustomWorld, user1: string, _user2: string) {
     const user1Parsed = worldUsers.parse(user1);
     const user1Profile = profiles[user1Parsed];
-    worldUsers.parse(user2);
     const zaakNumber = this.testStorage.get("caseNumber");
 
-    checkZaakAssignment.call(this, zaakNumber, user1Profile);
+    await checkZaakAssignment.call(this, zaakNumber, user1Profile);
   },
 );
 
 Then(
-  "Employee {string} sees the task assigned to Employee {string} in my task list",
-  { timeout: 120 * 1000 },
-  async function (this: CustomWorld, user1, user2) {
+  "Employee {string} sees the task assigned to them by Employee {string} in my task list",
+  { timeout: TWO_MINUTES_IN_MS },
+  async function (this: CustomWorld, user1: string, _user2: string) {
     const user1Parsed = worldUsers.parse(user1);
     const user1Profile = profiles[user1Parsed];
-    worldUsers.parse(user2);
 
     const caseNumber = this.testStorage.get("caseNumber");
 
     await this.page.goto(`${this.worldParameters.urls.zac}/taken/mijn`);
 
-    this.expect(
-      await this.page
-        .getByRole("cell", { name: caseNumber, exact: true })
-        .first(),
-    ).toBeTruthy();
-    this.expect(
-      await this.page
+    await this.expect(
+      this.page.getByRole("cell", { name: caseNumber, exact: true }).first(),
+    ).toBeVisible({ timeout: FIFTEEN_SECONDS_IN_MS });
+
+    await this.expect(
+      this.page
         .getByRole("cell", { name: "Aanvullende informatie", exact: true })
         .first(),
-    ).toBeTruthy();
-    // current date to 16-02-2024 format
-    const currentDDateString = new Date()
-      .toISOString()
-      .split("T")[0]
-      .split("-")
-      .reverse()
-      .join("-");
-    this.expect(
-      await this.page.getByRole("cell", { name: currentDDateString }).first(),
-    ).toBeTruthy();
-    this.expect(
-      await this.page.getByRole("cell", { name: user1Profile.group }).first(),
-    ).toBeTruthy();
+    ).toBeVisible();
+
+    // TODO: can't get this to work
```

Comment:
We should remove this check as discussed today. If we really need to check the format we can move do it in integration tests

Reply:
Agreed, thanks! I removed it.